generators/client-2/templates/webpack/_webpack.common.js

```diff
@@ -95,7 +95,21 @@ module.exports = function (options) {
             ]),
             new webpack.ProvidePlugin({
                 $: "jquery",
-                jQuery: "jquery"
+                jQuery: "jquery",
+                "window.jQuery": "jquery",
+                'Tether': 'tether',
+                'window.Tether': 'tether',
+                Tooltip: "exports?Tooltip!bootstrap/js/dist/tooltip",
```

Comment:
This configuration is needed because of the issue https://github.com/shakacode/bootstrap-loader/issues/172.  If I didn't include these changes, I should comment the bootstrap scripts from the _.bootstraprc file.